### ISSUE TITLE
Relationships are created even if the entity has been skipped

### ISSUE THREAD

**Issue by BackToTheCrm** [daryllabar/DLaB.Xrm.XrmToolBoxTools#19]
When generating only selected entities, the relationships to these entities are still generated.
For example, in Account.cs there is the following relationship to Invoice even if I skipped the Invoice entity\:
```
/// <summary>
/// 1:N invoice_customer_accounts
/// </summary>
[Microsoft.Xrm.Sdk.RelationshipSchemaNameAttribute("invoice_customer_accounts")]
public System.Collections.Generic.IEnumerable<Identicar.BtB.Common.Entities.Invoice> invoice_customer_accounts
{
	[System.Diagnostics.DebuggerNonUserCode()]
	get
	{
		return this.GetRelatedEntities<Identicar.BtB.Common.Entities.Invoice>("invoice_customer_accounts", null);
	}
	[System.Diagnostics.DebuggerNonUserCode()]
	set
	{
		this.OnPropertyChanging("invoice_customer_accounts");
		this.SetRelatedEntities<Identicar.BtB.Common.Entities.Invoice>("invoice_customer_accounts", null, value);
		this.OnPropertyChanged("invoice_customer_accounts");
	}
}
```

These relationships must not be generated otherwise it leads to compilation errors.

Also, it will be wonderful if you can add an option for disabling the generation of the relationships, indeed I never use them in my code.

**Comment by BackToTheCrm** [comment]
Update \: I can't reproduce the error, so I think it's a mistake coming from me.

However, it will still be wonderful if you can add an option for disabling the generation of the relationships.

**Comment by daryllabar** [comment]
The CrmSvcUtil should handle not creating the relationships of you are not selecting those entities to be generated.  I should be able to add the ability to skip generating relationship properties.

**Comment by BackToTheCrm** [comment]
Indeed, it works as you described.

**Comment by daryllabar** [comment]
Try this to see if it works [Plugin.zip](https://github.com/daryllabar/DLaB.Xrm.XrmToolBoxTools/files/143220/Plugin.zip).  Just uncheck the Generate Entity Relationships before generating your entities.

**Comment by BackToTheCrm** [comment]
It perfectly works !

**Comment by daryllabar** [comment]
Now included in 2016.2.23.2
